docs/roadmap.markdown

```diff
@@ -4,9 +4,172 @@ title: "Roadmap"
 permalink: /roadmap/
 ---
 
-The Developer Toolkit is comprised of a set of tools that are worked on in
-collaboration with the community.  The following unified roadmap provides an
-overview of the current priorities across the toolkit, and links to the
-specific tool roadmaps.
+## Introduction
 
+This is a community driven project, sponsored by New Relic, to build a
+community around Observability as Code with New Relic. The following is the
+initial work on defining the problem space, whom is impacted, and a roadmap for
+success.
+
+### Get Involved
+
+In the future, we will establish a [Request for Comments](https://en.wikipedia.org/wiki/Request_for_Comments) process
+as the channel for driving this roadmap.  For now we are using GitHub as the
```

Comment:
Try this instead:   "Currently, we are using GitHub to communicate with the community and represent a source of truth. Please use the following tools to contribute:"